### ISSUE TITLE
The GPU card lists supported by Apollo D-Kit hardware platform with Apollo git master version installed.

### ISSUE THREAD

**Issue by hongyi-zhao** [ApolloAuto/apollo#12708]
Hi,
 
On Apollo hardware platform numbered as 2020012(4), the host OS is Ubuntu 20.04 with Apollo git master version installed. For this case, what GPUs can be supported?

Regards,
HY

**Comment by oska874** [comment]
It's okay. You can try it.

**Comment by hongyi-zhao** [comment]
Nvidia has too many GPU series, so I want to see a specific list supported by the latest Apollo git master version. I've greped the source code and still can't find the descriptions on this topic.

**Comment by oska874** [comment]
Rtx 10xx,20xx almost okay.30xx havenot tested.

**Comment by hongyi-zhao** [comment]
What about [NVIDIA TITAN RTX](https://www.nvidia.com/en-us/deep-learning-ai/products/titan-rtx/) 24GB version?

**Comment by oska874** [comment]
It's a turing gpu like rtx 20xx.You need to check your power apply and budget.

获取 Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by storypku** [comment]
Apollo master as of today (Nov 13 of 2020) support common NVidia GPU cards, with compute capability from 3.7, 5.0, 6.0, 6.1 up-to 7.5 now. Support for Ampere GPU ( 8.0 and 8.6) is WIP.

**Comment by storypku** [comment]
> > Rtx 10xx,20xx almost okay.30xx havenot tested.
> 
> What about [NVIDIA TITAN RTX](https://www.nvidia.com/en-us/deep-learning-ai/products/titan-rtx/) 24GB version?
> 
> NB: According to the notes [here](https://github.com/ApolloAuto/apollo#prerequisites): NVIDIA Turing GPU is strongly recommended. This is inconsistent with your above assertion.

TITAN RTX is shipped with CUDA compute capability 7.5. And is supported since Apollo 6.0.

**Comment by hongyi-zhao** [comment]
> TITAN RTX is shipped with CUDA compute capability 7.5. And is supported since Apollo 6.0.

What's the correspondence between the versions of  CUDA and CUDA compute capability? Say, for my case, I can only see the  CUDA version shown as below:

```
werner@X10DAi:~$ nvidia-smi 
Fri Nov 13 19:34:15 2020       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 440.64       Driver Version: 440.64       CUDA Version: 10.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce RTX 207...  Off  | 00000000:02:00.0  On |                  N/A |
|  0%   40C    P8    19W / 215W |    250MiB /  7977MiB |      2%      Default |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    0      1842      G   /usr/lib/xorg/Xorg                            35MiB |
|    0      2422      G   /usr/lib/xorg/Xorg                            71MiB |
|    0      3437      G   /usr/bin/gnome-shell                          90MiB |
|    0      3894      G   ...tx-baidupinyin/files/bin/baidu-qimpanel    36MiB |
|    0    173298      G   ...lic/browser/mozilla/firefox/firefox-bin     2MiB |
+-----------------------------------------------------------------------------+
```

Regards,
HY

**Comment by storypku** [comment]
> > TITAN RTX is shipped with CUDA compute capability 7.5. And is supported since Apollo 6.0.
> 
> What's the correspondence between the versions of CUDA and CUDA compute capability? Say, for my case, I can only see the CUDA version shown as below:
> 
> ```
> werner@X10DAi:~$ nvidia-smi 
> Fri Nov 13 19:34:15 2020       
> +-----------------------------------------------------------------------------+
> | NVIDIA-SMI 440.64       Driver Version: 440.64       CUDA Version: 10.2     |
> |-------------------------------+----------------------+----------------------+
> | GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
> | Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
> |===============================+======================+======================|
> |   0  GeForce RTX 207...  Off  | 00000000:02:00.0  On |                  N/A |
> |  0%   40C    P8    19W / 215W |    250MiB /  7977MiB |      2%      Default |
> +-------------------------------+----------------------+----------------------+
>                                                                                
> +-----------------------------------------------------------------------------+
> | Processes:                                                       GPU Memory |
> |  GPU       PID   Type   Process name                             Usage      |
> |=============================================================================|
> |    0      1842      G   /usr/lib/xorg/Xorg                            35MiB |
> |    0      2422      G   /usr/lib/xorg/Xorg                            71MiB |
> |    0      3437      G   /usr/bin/gnome-shell                          90MiB |
> |    0      3894      G   ...tx-baidupinyin/files/bin/baidu-qimpanel    36MiB |
> |    0    173298      G   ...lic/browser/mozilla/firefox/firefox-bin     2MiB |
> +-----------------------------------------------------------------------------+
> ```
> 
> Regards,
> HY

Apollo Issues is the place for issues closely related to Apollo, not a place for general discussions. 
You should find answers on general questions yourself.

Take this one for example, you should check it on https://developer.nvidia.com/cuda-gpus#compute .
